shipit-engine.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency 'rails', '~> 4.2.2'
+  s.add_dependency 'rails', '~> 4.2.0'
```

Comment:
If it is worth noting, 4.2.1 is vulnerable to some security issues and you
should not be using. Even more in a new application.

On Fri, Jul 17, 2015, 13:26 Chris McKnight <notifications@github.com> wrote:

> In shipit-engine.gemspec
> <https://github.com/Shopify/shipit-engine/pull/459#discussion_r34905683>:
>
> > @@ -16,7 +16,7 @@ Gem::Specification.new do |s|
> >    s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
> >    s.test_files = Dir["test/**/*"]
> >
> > -  s.add_dependency 'rails', '~> 4.2.2'
> > +  s.add_dependency 'rails', '~> 4.2.0'
>
> I ran into a conflict because I ran rails new with Rails 4.2.1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Shopify/shipit-engine/pull/459/files#r34905683>.
>